projects/plugins/boost/app/modules/cache/pre-wordpress/Boost_Cache_Utils.php

```diff
@@ -5,15 +5,20 @@
 
 namespace Automattic\Jetpack_Boost\Modules\Page_Cache\Pre_WordPress;
 
+define( 'JBCACHE_ALL', 1 ); // delete all files and directories in a given directory, recursively.
+define( 'JBCACHE_FILE', 2 ); // delete a single file or recursively delete a single directory in a given directory.
+define( 'JBCACHE_FILES', 3 ); // delete all files in a given directory.
```

Comment:
Can we use strings as value, they will be easier to understand when logging? It might be a good idea to also move them to be class constants to keep them isolated and within Boost_Cache_Utils.